generated/java/gapic-google-cloud-texttospeech-v1beta1/build.gradle

```diff
@@ -34,4 +34,3 @@ sourceSets {
 }
 
 compileJava.options.encoding = "UTF-8"
-javadoc.options.encoding = "UTF-8"
```

Comment:
We need to retain this line. @vam-google added it manually in https://github.com/googleapis/api-client-staging/pull/621 and is adding it to toolkit in https://github.com/googleapis/toolkit/pull/1894/files#diff-6b8cabd49d503be9cdd76bb9d5512b2a which isn't submitted yet.

Reply:
The toolkit PR is merged